components/pluralrules/tests/plurals.rs

```diff
@@ -1,10 +1,13 @@
+use icu_cldr_json_data_provider::{CldrJsonDataProvider, CldrPaths};
 use icu_locale::LanguageIdentifier;
-use icu_pluralrules::data::provider::DummyDataProvider;
 use icu_pluralrules::{PluralCategory, PluralRuleType, PluralRules};
 
 #[test]
 fn test_plural_rules() {
-    let dp = DummyDataProvider::default();
+    let mut paths = CldrPaths::default();
+    paths.cldr_core = Ok("./data/cldr-core".into());
```

Comment:
Where does `./data/cldr-core` come from?

For testing, I would suggest copying [components/fs-data-provider/tests/testdata/json_plurals_37](https://github.com/unicode-org/icu4x/tree/master/components/fs-data-provider/tests/testdata/json_plurals_37) into components/pluralrules/tests/testdata/json_plurals_37, and then reading it with FsDataProvider.

Reply:
I generated data for all files instead, since my test harness uses locales not in your sample.

Maybe during reorg we could unify our data fixtures for all components into a single location and reference from all crates tests/benchmarks/examples.